### PR TITLE
Followup edits to 4814

### PR DESCRIPTION
**Pull request by bfallonf**
#4814 

cc @pecameron 

@openshift/team-documentation PTAL

### CODE REVIEW

File: architecture/core_concepts/routes.adoc [hunk 1]
```diff
@@ -179,31 +179,35 @@ image::router_model.png[HAProxy Router Data Flow]
...
-by using the `*haproxy.router.openshift.io/balance*` xref:route-specific-annotations[route specific annotation].
-
-Other types of routes use the `leastconn` xref:load-balancing[load balancing strategy] by
-default.  The default can be changed by using the `*ROUTER_LOAD_BALANCE_ALGORITHM*`
-xref:env-variables[environment variable]. It can be changed for individual routes
-by using the `*haproxy.router.openshift.io/balance*` xref:route-specific-annotations[route specific annotation].
+Sticky sessions ensure that all traffic from a user's session go to the same
+pod, creating a better user experience. While satisfying the user's requests,
+the pod caches data, which can be used in subsequent requests. For example, for
+a cluster with five back-end pods, and two load-balanced routers, you can ensure
```
Comment (mburke5678) [inline architecture/core_concepts/routes.adoc]: Not sure you need a comma after "pods"

File: architecture/core_concepts/routes.adoc [hunk 2]
```diff
@@ -179,31 +179,35 @@ image::router_model.png[HAProxy Router Data Flow]
...
+that the same pod receives the web traffic from the same web browser regardless
+of the router that handles it.
+
+While returning routing traffic to the same pod is desired, it cannot be
+guaranteed. However, you can use HTTP headers to set a cookie to determine the
+pod used in the last connection. When the user sends another request to the
+application the browser re-sends the cookie and the router knows where to send
+the traffic.
+
+Cluster administrators, you can turn off stickiness for passthrough routes
```
Comment (mburke5678) [inline architecture/core_concepts/routes.adoc]: s/administrators, you can/administrators can

File: architecture/core_concepts/routes.adoc [hunk 3]
```diff
@@ -179,31 +179,35 @@ image::router_model.png[HAProxy Router Data Flow]
...
+By default, sticky sessions for passthrough routes are implemented using the
+`source` xref:load-balancing[load balancing strategy]. The default can be
+changed for all passthrough routes by using the `*ROUTER_TCP_BALANCE_SCHEME*`
+xref:env-variables[environment variable]. It can be changed for individual
+routes by using the `*haproxy.router.openshift.io/balance*`
+xref:route-specific-annotations[route specific annotation].
+
+Other types of routes use the `leastconn` xref:load-balancing[load balancing
+strategy] by default, which can be changed by using the
+`*ROUTER_LOAD_BALANCE_ALGORITHM*` xref:env-variables[environment variable]. It
```
Comment (mburke5678) [inline architecture/core_concepts/routes.adoc]: What is "It"? Load balancing strategy?

**Comment by mburke5678** [comment]
@bfallonf Couple of nits. Otherwise LGTM

File: architecture/core_concepts/routes.adoc (hunk 3 above)
Comment (bfallonf) [inline architecture/core_concepts/routes.adoc]: Changed the wording to be more specific.

**Comment by bfallonf** [comment]
Thanks @mburke5678 . Fixed as suggested. I'll merge.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../architecture/core_concepts/routes.adoc#architecture-core-concepts-routes[Core Concepts -> Routes]
|Added the xref:../architecture/core_concepts/routes.adoc#load-balancing[Load-balancing Strategy] section, as well as some minor rewordings.
%

**Pull request merged**